File: messaging/v1/alpha_sender/create-default/create-default.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using Twilio;
using Twilio.Rest.Messaging.V1.Service;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var alphaSender = AlphaSenderResource.Create(
            alphaSender: "AlphaSender",
            pathServiceSid: "MGXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
        );

        Console.WriteLine(alphaSender.Sid);
    }
}

File: messaging/v1/phone_number/create-default/create-default.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

phone_number = @client.messaging
                      .services('MGXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX')
                      .phone_numbers
                      .create(
                         phone_number_sid: 'PNXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
                       )

puts phone_number.sid
